src/pre_defined_classes/mathematical_program/medium/q5/TrigValTest.java

```diff
@@ -0,0 +1,36 @@
+package pre_defined_classes.mathematical_program.medium.q5;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TrigValTest extends BaseTest {
+
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The value of tan\\^2 is\\: "),
+                new DoubleLiteral(2, 4, "tan2"),
+                new NewLine(),
+                new StringLiteral("The value of sec\\^2 is\\: "),
+                new DoubleLiteral(3, 5, "sec2"),
+                new NewLine(),
+                new StringLiteral("Their sum is\\: "),
+                new DoubleLiteral(6, 8, "sum")
+        };
+    }
+
+    public void runMain() {
+        TrigVal.main(new String[0]);
+    }
+
+    @Test
```

Comment:
Again, since there are no distractors, I don't see how the math could be wrong. It would be more useful to tell them how their order should be.

Reply:
It won't pass the clause test anyways if the format is wrong, so we should just provide the output. Also, I was told to check the math even if it was a Parsons question